src/components/MovieSlider.tsx

```diff
@@ -0,0 +1,151 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  Stack,
+  Tab,
+  TabList,
+  Tabs,
+} from "@chakra-ui/react";
+import MovieCard from "@/components/MovieCard";
+import React from "react";
+
+interface TabsOptions {
+  key: string;
+  title: string;
+}
+
+interface HeadingMovieSliderProps {
+  title: string;
+  tabOptions: TabsOptions[];
+  onTabChange: (index: number) => void;
+}
+
+const HeadingMovieSlider = ({
+  title,
+  tabOptions,
+  onTabChange,
+}: HeadingMovieSliderProps) => {
+  return (
+    <Stack
+      p={6}
+      flexDirection={["column", "row"]}
+      alignItems={["start", "center"]}
+      spacing={[4, 0]}
+    >
+      <Heading as={"h6"} fontSize={"2xl"} mr={4}>
+        {title}
+      </Heading>
+      <Tabs
+        variant="soft-rounded"
+        colorScheme="blue"
+        borderRadius={"full"}
+        border={"1px"}
+        height={"8"}
+        onChange={(index) => onTabChange(index)}
```

Comment:
ini bisa disingkat jadi gini : 

```suggestion
        onChange={onTabChange}
```

Reply:
Oiyaa yaa, sebenarnya `onTabChange` gak nerima parameter